ha/setup/ha_setup.py

```diff
@@ -632,7 +632,21 @@ def process(self):
         """
         Process reset command.
         """
-        pass # TBD
+        Log.info("Processing reset command")
+        # list of log folders
+        logs = [const.RA_LOG_DIR, const.PACEMAKER_LOG, const.PCSD_LOG]
+        for file in os.listdir(const.COROSYNC_LOG):
+            logs.append(os.path.join(const.COROSYNC_LOG, file))
+        # Delete the logs
+        self.remove_logs(logs)
+        Log.info("reset command is successful")
+
+    def remove_logs(self, logs: list):
+        """
+        Remove logs created by ha.
+        """
+        for log in logs:
+            ResetCmd.remove_file(log)
```

Comment:
In cleanup, services are still running. If log files are simply deleted then services may not be able to write to log file. I guess, you need to truncate the files and then send SIGHUP signal to syslog server. Please check.

Reply:
Applied logrotate to compress the older version of log file & created new log files. 
compressed log files are deleted.